synapse/models/infotech.py

```diff
@@ -1882,7 +1882,8 @@ def getModelDefs(self):
                         'disp': {'hint': 'text'},
                         'doc': 'The commit message describing the changes in the commit.'}),
 
-                    ('id', ('str', {}), {
+                    # we mirror the interface type options...
+                    ('id', ('str', {'strip': True}), {
```

Comment:
This will require a model migration.

Reply:
Nope. The interface specifies strip which used to take precedence.